packages/pyright-internal/typeshed-fallback/stdlib/tkinter/__init__.pyi

```diff
@@ -3294,7 +3294,7 @@ class LabelFrame(Widget):
         highlightcolor: _Color = ...,
         highlightthickness: _ScreenUnits = ...,
         # 'ne' and 'en' are valid labelanchors, but only 'ne' is a valid _Anchor.
-        labelanchor: Literal["nw", "n", "ne", "en", "e", "es", "se", "s", "sw", "ws", "w", "wn"] = ...,
+        labelanchor: Literal["nw", "n", "ne", "en", "e", "es", "se", "s", "sw", "ws", "w", "wn", "lol"] = ...,
```

Comment:
This and the following changes are to some unrelated type stubs (I think they're compass directions for UI layout!).

You need to add a case to localize.ts.

Reply:
https://github.com/microbit-foundation/pyright/pull/32/commits/8e1a7976e00c4250c6d9f2e40e8701e7219864dd